docker-compose.yml

```diff
@@ -87,36 +87,20 @@ services:
       ca.mcgill.a11y.image.preprocessor: 2
       ca.mcgill.a11y.image.port: 5000
 
-# object-detection service running for production and default profile - single GPU
   object-detection:
-    profiles: [production, default]
+    profiles: [production, default, test]
     image: ghcr.io/shared-reality-lab/image-preprocessor-object-detection-v8:${REGISTRY_TAG}
     restart: unless-stopped
     labels:
       ca.mcgill.a11y.image.preprocessor: 3
       ca.mcgill.a11y.image.port: 5000
     deploy:
-          resources:
-            reservations:
-              devices:
-              - driver: nvidia
-                capabilities: ["gpu", "utility", "compute"]
-
-# object-detection service running for test profile - multiple GPU
-  object-detection-test:
-    profiles: [test]
-    image: ghcr.io/shared-reality-lab/image-preprocessor-object-detection-v8:${REGISTRY_TAG}
-    restart: unless-stopped
-    labels:
-      ca.mcgill.a11y.image.preprocessor: 3
-      ca.mcgill.a11y.image.port: 5000
-    deploy:
-          resources:
-            reservations:
-              devices:
-              - driver: nvidia
-                device_ids: ['1']
-                capabilities: ["gpu", "utility", "compute"]
+      resources:
+        reservations:
+          devices:
+          - driver: nvidia
+            device_ids: ['0']
```

Comment:
I think tying it to a specific GPU can be ommitted entirely, which may mean it can intelligently assign it. If true, suggest removing the specific assignment in the core docker-compose.yml